Resources/Maps/shoukou.yml

```diff
@@ -28727,6 +28720,14 @@ entities:
       rot: 1.5707963267948966 rad
       pos: 6.5,-41.5
       parent: 34
+    - type: DeviceLinkSource
+      linkedPorts:
+        4263:
+        - MedicalScannerSender: CloningPodReceiver
+        - CloningPodSender: CloningPodReceiver
+        12439:
+        - MedicalScannerSender: MedicalScannerReceiver
+        - CloningPodSender: MedicalScannerReceiver
```

Comment:
These look linked wrong. When you're linking, link the computer to the scanner MedicalScannerSender->MedicalScannerSender. Then link the computer to the pod CloningPodSender->CloningPodSender.

(moved down for better clarity)

Reply:
I mean, I link the computers exactly how I know to make sure that they actualy work.